python/pyspark/ml/feature.py

```diff
@@ -760,6 +765,60 @@ class StringIndexerModel(JavaModel):
     """
     Model fitted by StringIndexer.
     """
+    @property
+    def labels(self):
+        return self._java_obj.labels
+
+
+class IndexToString(JavaTransformer, HasInputCol, HasOutputCol):
+    """
+    Convert provided indexes back to strings using either the metadata on the input column
+    or user provided labels.
+    Note: By default we keep the original columns during StringIndexerModel's transformation,
+    so the inverse should only be used on new columns such as predicted labels.
+    """
+    # a placeholder to make the labels show up in generated doc
+    labels = Param(Params._dummy(), "lables",
+                   "Optional labels to be provided by the user, if not supplied column " +
+                   "metadata is read for labels. The default value is an empty array, " +
+                   "but the empty array is ignored and column metadata used instead.")
+
+    @keyword_only
+    def __init__(self, inputCol=None, outputCol=None, labels=[]):
+        """
+        Initialize this instace of the IndexToString using the provided java_obj.
```

Comment:
first line should be:
```
__init__(self, inputCol=None, outputCol=None, labels=[])
```
as in other transformers (See VectorAssembler)

typo: instace